modules/OCPP201/OCPP201.cpp

```diff
@@ -864,6 +915,15 @@ void OCPP201::process_transaction_finished(const int32_t evse_id, const int32_t
             this->charge_point->on_charging_state_changed(evse_id, ocpp::v201::ChargingStateEnum::EVConnected,
                                                           ocpp::v201::TriggerReasonEnum::StopAuthorized);
         }
+    } else {
+        // TODO(piet): If StopTxOnEVSideDisconnect is false, authorization shall still be present. This cannot only be
```

Comment:
Is this for later?

Reply:
yes, cant be handled here and requires change to EvseManager and Auth